cmd/devcli/commands/healthcheck/healthcheck.go

```diff
@@ -0,0 +1,200 @@
+package healthcheck
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/service/lambda"
+	lambdaTypes "github.com/aws/aws-sdk-go-v2/service/lambda/types"
+	"github.com/briandowns/spinner"
+	"github.com/common-fate/clio"
+	"github.com/common-fate/clio/clierr"
+	"github.com/common-fate/common-fate/pkg/cfaws"
+	"github.com/common-fate/common-fate/pkg/config"
+	"github.com/common-fate/common-fate/pkg/deploy"
+	"github.com/common-fate/common-fate/pkg/pdk"
+	"github.com/common-fate/common-fate/pkg/service/healthchecksvc"
+	"github.com/common-fate/common-fate/pkg/types"
+	"github.com/common-fate/ddb"
+	"github.com/joho/godotenv"
+	"github.com/sethvargo/go-envconfig"
+	"github.com/urfave/cli/v2"
+)
+
+// this command can be run in dev with:
+// go run cf/cmd/cli/main.go healthcheck
+
+var Command = cli.Command{
+	Name:        "healthcheck",
+	Description: "healthcheck a deployment",
+	Usage:       "healthcheck a deployment",
+	Flags:       []cli.Flag{&cli.StringSliceFlag{Name: "deployment-mappings"}},
+	Subcommands: []*cli.Command{
+		&LocalCommand,
+		&LambdaCommand,
+	},
+}
+
+var LocalCommand = cli.Command{
```

Comment:
Update this to handle API responses correctly and include instructions in the readme or command usage on how to use the dev overrides